grpc/proto/space.proto

```diff
@@ -27,13 +27,6 @@ service SpaceApi {
     };
   }
 
-  // Get Config information installed daemon
-  rpc GetConfigInfo(google.protobuf.Empty) returns (ConfigInfoResponse) {
```

Comment:
Also, don't forget to delete the 
```
message ConfigInfoResponse {
  string port = 1;
  string appPath = 2;
}
```

Reply:
Fixed in 9de527b.
Will merge soon if we don't get additional comments.